app/views/layouts/modals/_kindergarten_results_aborted.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="modal fade mu-kids-results-aborted mu-kindergarten-context" id="kids-results-aborted" tabindex="-1" role="dialog" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content danger">
+      <button type="button" class="mu-kindergarten-modal-button mu-close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">
+              <i class="fa fa-fw fa-times fa-3x"></i>
+            </span>
+      </button>
+      <h4 class="modal-header text-broken">
+        <strong><%= t :errored %></strong>
```

Comment:
:warning: There are multiple issues with these file: 

1. It is called `aborted`, but it has the `errored` state hardcoded
2. It has the `errored` state and `broken` class, but it is used even when tests fail. 

This leads to the following situation when a test fails normally: 

![image](https://user-images.githubusercontent.com/677436/98131450-09043680-1e9a-11eb-940a-d7dfdebf60c8.png)